x-pack/legacy/plugins/infra/public/containers/logs/log_highlights/log_highlights.tsx

```diff
@@ -28,16 +26,14 @@ export const useLogHighlightsState = ({
   filterQuery: string | null;
 }) => {
   const [highlightTerms, setHighlightTerms] = useState<string[]>([]);
-  const { visibleMidpoint, jumpToTargetPosition } = useContext(LogPositionState.Context);
-  const { intervalSize: summaryIntervalSize } = useContext(LogViewConfiguration.Context);
   const {
-    start: summaryStart,
-    end: summaryEnd,
-    bucketSize: summaryBucketSize,
-  } = useLogSummaryBufferInterval(
```

Comment:
We no longer need `summary*` values for the range. We just use the `*Timestamp` values.

The `bucketSize` is calculated dynamically inside the `logSummary` and `logSummaryHighlights` hook, so we also don't need to pass it anymore.